clusterloader2/testing/load/config.yaml

```diff
@@ -100,8 +138,8 @@ steps:
     Params:
       kind: ReplicationController
       labelSelector: kind = loadRC
-      # timeout = (totalPodsRounded /20)s + 3m
-      timeout: {{DivideInt $totalPodsRounded 20 | AddInt 180}}s
+      # timeout = (totalPods /20)s + 3m
```

Comment:
This is also a thing that we need to rethink how to do.
The way how it is done now is:
- you create everything (in the previous step), spreaded over extended period of time, potentially hours
- only then you actually verify if pods were starting (so for some RC, the timeout may be hours, not really what you have below).

That is clearly not what we want. What we probably want is:
- initialize WaitForControlledPodsRunning
- it will be watching RCs and for each of them will be spawning goroutine watching its pods
- these go-routines may finish (with success or failure or timeout)
- the at the end you gather results of this measurement
[This gathering is synchronous and make take time]

But that seems to be the only reasonable way to do it.

Reply:
#264 - Reimplementing WaitForControlledPodsRunning